mywebsite/news/templates/news/base.html

```diff
@@ -41,7 +43,7 @@ <h3>Самые обсуждаемые новости:</h3>
                 </li>
                 {% endfor %}
                 <hr>
-                <a href="{% url 'news:article_list' %}">Список всех новостей</a>
+                <a href="{% url 'django.sontrib.sitemaps.views.sitemap' %}">XML-карта сайта</a>
```

Comment:
Можно сделать проще)

Reply:
@udevgeektutor, каким образом?

Reply:
изучить документацию функции path, просмотреть прошлые примеры использования в проекте.
+тут явно чтото не то, похоже и на опечатку и на избыточность.
во вьюшке ты импортируешь sitemap, зачем тут писать для контекста передавать такое сложное имя как строку ---> 'django.sontrib.sitemaps.views.sitemap'?
```suggestion
                <a href="{% url 'django.sontrib.sitemaps.views.sitemap' %}">XML-карта сайта</a>
```

Reply:
А такое разве существет?
'django.sontrib'

Reply:
до шаблона код можно донести хотя бы так
```suggestion
                <a href="{% url 'sitemaps' %}">XML-карта сайта</a>
```

Reply:
естественн подогнав вьюшку под это)

Reply:
Не понимаю о чём речь. Если дело лишь в том, что нужно исправить опечатку в name, а лучше вообще сократить name - хорошо, сделаю. Так было в лекции (если не считать опечатки), а дальше я лишь сослался на нужный url. О каком подгоне вьюшки идёт речь - не понимаю: указанный код является частью base, который используется на всех вьюшках, а вьюшка для sitemap импортируется прямо в urls.py из числа встроенных

Reply:
@udevgeektutor, сделано.